Description: change default values for debian
Author: Jonas Genannt <jonas.genannt@capi2name.de>
Forwarded: not-needed


diff --git a/conf/carbon.conf.example b/conf/carbon.conf.example
index 94d9185..1e57845 100644
--- a/conf/carbon.conf.example
+++ b/conf/carbon.conf.example
@@ -14,29 +14,21 @@
 # To change other directory paths, add settings to this file. The following
 # configuration variables are available with these default values:
 #
-#   STORAGE_DIR    = $GRAPHITE_STORAGE_DIR
-#   LOCAL_DATA_DIR = STORAGE_DIR/whisper/
-#   WHITELISTS_DIR = STORAGE_DIR/lists/
-#   CONF_DIR       = STORAGE_DIR/conf/
-#   LOG_DIR        = STORAGE_DIR/log/
-#   PID_DIR        = STORAGE_DIR/
-#
-# For FHS style directory structures, use:
-#
-#   STORAGE_DIR    = /var/lib/carbon/
-#   CONF_DIR       = /etc/carbon/
-#   LOG_DIR        = /var/log/carbon/
-#   PID_DIR        = /var/run/
-#
-#LOCAL_DATA_DIR = /opt/graphite/storage/whisper/
+
+STORAGE_DIR    = /var/lib/graphite/
+CONF_DIR       = /etc/carbon/
+LOG_DIR        = /var/log/carbon/
+PID_DIR        = /var/run/
+
+LOCAL_DATA_DIR = /var/lib/graphite/whisper/
 
 # Enable daily log rotation. If disabled, a kill -HUP can be used after a manual rotate
-ENABLE_LOGROTATION = True
+ENABLE_LOGROTATION = False
 
 # Specify the user to drop privileges to
 # If this is blank carbon runs as the user that invokes it
 # This user must have write access to the local data directory
-USER =
+USER = _graphite
 #
 # NOTE: The above settings must be set under [relay] and [aggregator]
 #       to take effect for those daemons as well
