Author: Valentin Haenel <valentin.haenel@gmx.de>
Date: Wed, 13 Nov 2013 15:33:40 +0100
Description: [PATCH] remove daemonize for compatability with twisted 13.2.0
Origin: https://github.com/ImmobilienScout24/carbon/commit/1d0bdc5c6ac331fdcb6f6c0f897507fcf833f616

--- a/lib/carbon/util.py
+++ b/lib/carbon/util.py
@@ -18,10 +18,6 @@ except:
 from time import sleep, time
 from twisted.python.util import initgroups
 from twisted.scripts.twistd import runApp
-from twisted.scripts._twistd_unix import daemonize
-
-
-daemonize = daemonize  # Backwards compatibility
 
 
 def dropprivs(user):
